Dockerfile

```diff
@@ -5,7 +5,8 @@ RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 
 # Install system commands, Android SDK, and Ruby
 RUN apt-get update  \
-    && apt-get install -y coreutils git wget locales android-sdk android-sdk-build-tools \
+    && apt-get install -y coreutils git wget locales android-sdk android-sdk-build-tools bzip2 \
+	&& apt-get install -y autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm6 libgdbm-dev libdb-dev \
```

Comment:
This is the [suggested build environment](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment) for `rbenv`. 
A couple of packages are already installed in `Debian`, but since there's no arm with keeping all of them in the command line, I preferred to stick with the complete set.